docs/section5/Rule5-37.md

```diff
@@ -0,0 +1,81 @@
+
+    - Else, request manual review: `else: manual_review_flag = TRUE`
+
+  - Else, building segment does not have exterior mixed type roof surface: `else:`
+
+    - Get skylight-roof-ratio for residential type roofs: `srr_res = skylight_roof_ratios_dictionary_b[building_segment_b.id]["EXTERIOR RESIDENTIAL"]`
+
+      - If skylight-roof-ratio is greater than 2.0%, get baseline skylight construction requirement: `if srr_res > 0.02: target_u_factor_res = data_lookup(table_G3_4, climate_zone, "RESIDENTIAL", "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U")`
+
+      - Else, skylight-roof-ratio is 0% to 2.0%, get baseline skylight construction requirement: `else: target_u_factor_res = data_lookup(table_G3_4, climate_zone, "RESIDENTIAL", "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U")`
+
+    - Get skylight-roof-ratio for non-residential type roofs: `srr_nonres = skylight_roof_ratios_dictionary_b[building_segment_b.id]["NON-RESIDENTIAL"]`
+
+      - If skylight-roof-ratio is greater than 2.0%, get baseline skylight construction requirement: `if srr_nonres > 0.02: target_u_factor_nonres = data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL", "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U")`
+
+      - Else, skylight-roof-ratio is 0% to 2.0%, get baseline skylight construction requirement: `else: target_u_factor_nonres = data_lookup(table_G3_4, climate_zone, "NON-RESIDENTIAL", "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U")`
+
+  - Get skylight-roof-ratio for semi-exterior type roofs: `srr_semi_exterior = skylight_roof_ratios_dictionary_b[building_segment_b.id]["SEMI-EXTERIOR"]`
+
+    - If skylight-roof-ratio is greater than 2.0%, get baseline skylight construction requirement: `if srr_semi_exterior > 0.02: target_u_factor_semiheated = data_lookup(table_G3_4, climate_zone, "SEMIHEATED, "SKYLIGHT", "2.1%+", "ASSEMBLY MAX. U")`
+
+    - Else, skylight-roof-ratio is 0% to 2.0%, get baseline skylight construction requirement: `else: target_u_factor_semiheated = data_lookup(table_G3_4, climate_zone, "SEMIHEATED", "SKYLIGHT", "0%-2.0%", "ASSEMBLY MAX. U")`
+
+  - For each zone in building segment: `for zone_b in building_segment_b...zones:`
+
+    - For each surface in zone: `for surface_b in zone_b.surfaces:`
+
+      - Check if surface is roof and has subsurface: `if ( get_opaque_surface_type(surface_b) == "ROOF"  ) AND ( surface_b.subsurfaces ):`
+
```

Comment:
We need to check whether subsurface.classification!= "door" OR that subsurface.classification== "door" AND subsurface.glazed_area>subsurface.opaque_area. In other words, we need to update the check so that it is not applies to doors that is 50% or less glass. The door counts as fenestration if it's more than 50% glass.

Reply:
Updated.